src/basic/conf-files.c

```diff
@@ -200,7 +196,7 @@ int conf_files_insert(char ***strv, const char *root, char **dirs, const char *p
         for (i = 0; i < strv_length(*strv); i++) {
                 int c;
 
-                c = base_cmp(*strv + i, &path);
+                c = base_cmp((char * const *) *strv + i, (char * const *) &path);
```

Comment:
I think we prefer to put the star flush against one of the words, so that this does not look like multiplication. I.e. `char* const*` in this case.

Reply:
OK.